circle.yml

```diff
@@ -47,6 +47,11 @@ deployment:
     commands:
       - cf login -a https://api.system.platform.digital.gov.au -o dto -u $CF_USER_PROD -p $CF_PASSWORD_PROD
       - cf target -o dto -s digital-marketplace
-      - cf push -f 'manifest.production.yml' -c 'python application.py db upgrade' -u 'none'
-      - cf push -f 'manifest.production.yml'
+      - cf unmap-route dm-api apps.platform.digital.gov.au --hostname dm-api
+      - cf push dm-api -f 'manifest.production.yml' -c 'python application.py db upgrade' -u 'none'
```

Comment:
Needs an "-i 1" to make sure we don't have two instances trying to do a migration.

Also, if we're going to have the migration running while the app is running, anyway, how about doing it before unmapping the route?  Also means that both apps will be left running normally if the migration fails.  As another bonus, the two pushes will be the same and can be put into a script like with the Buyer/Supplier frontends.